### PR TITLE
Stuff -  Added and fixed things yes

### PR DESCRIPTION
**Pull request by jani270**
Fixed a lot of NBT Tags
Fixed Hardstone Minion and other Items
Removed not used things from the repo 
Added Reforge Stats
Added Wiki Links
Added Featurelink

If you have question about the link read the conversation from [here](https://canary.discord.com/channels/516977525906341928/755816367827058749/880846573536411780)

### CODE REVIEW

File: constants/enchants.json [hunk 1]
```diff
@@ -396,101 +396,6 @@
       "turbo_wheat"
     ]
   ],
-  "enchants_min_level": {
```
Comment (IRONM00N) [inline constants/enchants.json:399]: Why?

File: constants/misc.json [hunk 2]
```diff
@@ -31,65 +31,6 @@
     "very special": "§c"
   },
 
-  "special_enchants": [
```
Comment (IRONM00N) [inline constants/misc.json:34]: also why?

File: constants/misc.json [hunk 3]
```diff
@@ -117,16 +58,6 @@
     100000
   ],
 
-  "mob_names": {
```
Comment (IRONM00N) [inline constants/misc.json:120]: Still why?

File: constants/misc.json [hunk 4]
```diff
@@ -144,124 +75,6 @@
     "crystal_hollows": "Crystal Hollows"
   },
 
-  "bag_size": {
```
Comment (IRONM00N) [inline constants/misc.json:147]: once again why?

**Comment by jani270** [comment]
i didnt do the format was just the site

File: constants/misc.json (hunk 3 above)
Comment (jani270) [reply constants/misc.json:120]: Not used in NEU at all

File: constants/misc.json (hunk 4 above)
Comment (jani270) [reply constants/misc.json:147]: Not used in NEU at all again

**Review by IRONM00N — APPROVED** [review]

**Pull request merged**